packages/integration-sdk/CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to
 
 ## Unreleased
 
+## 2.3.0 - 2020-07-22
+
+### Added
+
+- Disk usage metrics are now published by the logger.
+
+## 2.3.0 - 2020-07-22
```

Comment:
Should this line be removed?

Reply:
Ah, that should stay (the date should change) and the most recent header should change to 2.4.0.